examples/nextjs/src/app/api/auth/route.ts

```diff
@@ -2,10 +2,42 @@ import { CONNECTION_STRING } from '@/lib/config'
 import { getOrCreateDocAndToken } from '@y-sweet/sdk'
 import { NextResponse } from 'next/server'
 
+const ALLOWED_ORIGINS = new Set([
+  'demos.y-sweet.dev',
+  'blocknotejs.org',
+  'blocknote-main.vercel.app',
+  'localhost',
+])
```

Comment:
I'm tempted just to drop the hard-coded list of sites, and add a comment saying that in production you'd want to set `access-control-allow-origin` to a specific domain.

Reply:
I'm fine with that, it just means that anyone will be able to use `demos.y-sweet.dev` as their actual live Y-Sweet service.

If you give the go-ahead I'll make the change!

Reply:
Yeah, I'm fine with that, we already provide a generous free tier so they shouldn't have a reason to do that :)

Reply:
I think I'm more worried about people just copying/pasting/deploying to Vercel or something without ever really knowing what's going on. But I suppose those people are probably unlikely to convert anyway, and if a project like that goes viral we can cross that bridge when we get to it.

Reply:
If they deploy this codebase to vercel it will have its own instance of the auth route so it should just hit that, so I think we’re fine, someone would have to intentionally hard-code in the demo server domain to hit the demo server.